torch/_dynamo/utils.py

```diff
@@ -1529,16 +1530,6 @@ def tensor_always_has_static_shape(
     return False, None
 
 
-class LazyString:
-    def __init__(self, func, *args, **kwargs):
-        self.func = func
-        self.args = args
-        self.kwargs = kwargs
-
-    def __str__(self):
-        return self.func(*self.args, **self.kwargs)
```

Comment:
Just a small cleanup; this got moved to torch._logging. I actually didn't need it in this PR but I kept the cleanup. If you REALLY care I can split it out.